core/src/com/unciv/logic/trade/TradeType.kt

```diff
@@ -15,7 +15,9 @@ enum class TradeType(val numberType: TradeTypeNumberType, val isImmediate: Boole
     Luxury_Resource     (TradeTypeNumberType.Simple, false),
     Strategic_Resource  (TradeTypeNumberType.Simple, false),
     Technology          (TradeTypeNumberType.None, true),
-    Introduction        (TradeTypeNumberType.None, true),
```

Comment:
By the way, @Deprecated("message") works just fine on _individual_ enum Instances. Point moot if made moddable.

Reply:
I felt like "deprecated" was not the right term here, as the feature was essentially completely removed (Introductions could no longer be offered). The left-in code was just to convert old saves. But like you said, it does not really matter anyway since I will un-remove it and put it behind a modOption.